app/src/androidTest/java/com/github/h3lp3rs/h3lp/H3lpAppTest.kt

```diff
@@ -99,6 +104,33 @@ open class H3lpAppTest {
         GeneralLocationManager.set(locationManagerMock)
     }
 
+    /**
+     * Custom matcher to test error on TextInputLayout.
+     * See : https://stackoverflow.com/questions/38842034/how-to-test-textinputlayout-values-hint-error-etc-using-android-espresso
+     */
+     fun hasInputLayoutError(): Matcher<View> = object : TypeSafeMatcher<View>() {
```

Comment:
Why is this a function, can't it simply be a val (or lazy val)?
It always returns the same matcher.

Reply:
Nice catch! I only copied it from another class and didn't pay attention to it